framework/tlvf/tlvf.py

```diff
@@ -1006,14 +1006,14 @@ def addClassVarLenMethods(self, obj_meta, param_type, param_name, param_meta, pa
             lines_cpp.append( "%sreturn false;" % self.getIndentation(2))
             lines_cpp.append( "%s}" % self.getIndentation(1) )
 
-            lines_cpp.append("%suint8_t *src = (uint8_t*) m_%s;" % (self.getIndentation(1), param_meta.name))
-            lines_cpp.append("%sif (!m_parse__) {" %self.getIndentation(1))
+            lines_cpp.append("%suint8_t *src = (uint8_t *)m_%s;" % (self.getIndentation(1), param_meta.name))
```

Comment:
Remove space before `*`
Should be `(uint8_t*)m_%s;`

Reply:
space should be between the type to the asterisks.